tests/unit/amo/components/TestUserProfile.js

```diff
@@ -926,6 +926,15 @@ describe(__filename, () => {
     expect(root.find(NotFound)).toHaveLength(1);
   });
 
+  it('renders a user profile when URL contains a user ID', () => {
+    const { store } = signInUserWithUsername('black-panther');
+    const user = getCurrentUser(store.getState().users);
+
+    const root = renderUserProfile({ params: { username: user.id }, store });
+
+    expect(root.find('.UserProfile')).toHaveLength(1);
```

Comment:
It seems like this test should also be checking that the profile page that is rendered in fact belongs to the user who corresponds with the `id` used? It's probably also a good idea to check the properties passed into `Paginate` and the `AddonsByAuthorsCard`s because that logic has changed with this patch.